utils/check_repo.py

```diff
@@ -557,6 +559,83 @@ def check_models_are_tested(module, test_file):
     return failures
 
 
+def find_doc_tested_doc(documentation_test_file="utils/documentation_tests.txt"):
+    """Parse the content of test_file to detect what's in all_model_classes"""
+    # This is a bit hacky but I didn't find a way to import the test_file as a module and read inside the class
+    with open(os.path.join(documentation_test_file), "r", encoding="utf-8", newline="\n") as f:
+        content = f.read()
+    all_docs = re.findall(r"(?:docs/).*.mdx", content)
```

Comment:
The .* should be adapted to get the shortest match, otherwise you might catch several files inside of it.
```suggestion
    all_docs = re.findall(r"(?:docs/).*?.mdx", content)
```